DevelopGitHub/GitHubExtension/GitHubExtension.Notes.WebApi/Controllers/NoteController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Threading.Tasks;
+using System.Web.Http;
+using GitHubExtension.Notes.WebApi.Services;
+using GitHubExtension.Notes.WebApi.ViewModels;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Notes.WebApi.Controllers
+{
+    public class NoteController : ApiController
+    {
+        private INotesService noteService;
+
+        public NoteController(INotesService noteService)
+        {
+            this.noteService = noteService;
+        }
+
+        [Route("api/note/{noteId}")]
+        public async Task<IHttpActionResult> GetNote([FromUri] int noteId)
+        {
+            var note = await noteService.GetNote(noteId);
+            if (note == null)
+            {
+                return NotFound();
+            }
+            else
+            {
+                return Ok(note);
+            }
+        }
+
+        [Route("api/note")]
+        [HttpPost]
+        public async Task<IHttpActionResult> CreateNote([FromBody]AddNoteModel model)
```

Comment:
If you made POST operation all information automatically passed int the Body of http request.